src/__tests__/fixtures/json-schema/search/orders.json

```diff
@@ -0,0 +1,51 @@
+{
+	"title": "orders",
+	"additionalProperties": false,
```

Comment:
Is this the json generated? For search @himank this should always be true?

Reply:
Yes, this is the generated JSON being sent to the server. @himank should this always be true?

Reply:
Just remove it from here, and the server will automatically handle it for the search index. Let's not explicitly set it for any case.

Reply:
done